src/main/java/CarRacing/Domain/RacingGame.java

```diff
@@ -0,0 +1,31 @@
+package CarRacing.Domain;
+
+import java.util.List;
+
+public class RacingGame {
+    private final Cars cars;
+    private final Trial trial;
+
+    public RacingGame(final Cars cars, final Trial trial){
+        this.cars = cars;
+        this.trial = trial;
+    }
+
+    public void tryToMove(){
+        cars.tryToMove();
+        trial.moveToNextTrial();
+    }
+
+    public List<String> getWinners(){
```

Comment:
get을 쓰면 getter처럼 보일 수가 있습니다. 
```suggestion
    public List<String> findWinners(){
```
이게 좀 더 어울리지 않을까요? 추천해봅니다ㅎㅎ
[자바 메소드 명명 규칙](https://ozofweird.tistory.com/entry/Java-%EB%AA%85%EB%AA%85-%EA%B7%9C%EC%B9%99)을 두고 가니 한번 읽어보세요!